lib/game_state.rb

```diff
@@ -21,23 +22,22 @@ def player_move
 
       if error.nil?
         @board.update(current_player.mark, position)
+        @prev_move_position = position
         break
       end
 
-      display_board_with_messages_for_move bottom_messages: [error]
+      display_board_with_messages_for_move bottom_messages: [[error]]
     end
   end
 
-  def display_board_with_messages_with_welcome
-    display_board_with_messages_for_move top_message: :welcome
+  def display_board_with_messages_for_move(top_message: nil, bottom_messages: [])
+    bottom_messages = bottom_messages.push(move_instruction_message)
+    display_board_with_messages(top_message: top_message, bottom_messages: bottom_messages)
   end
 
-  def display_board_with_messages_for_move(top_message: :title, bottom_messages: [])
-    bottom_message_symbols = bottom_messages.push(instruction_symbol)
-    display_board_with_messages top_message: top_message, bottom_messages: bottom_message_symbols
-  end
+  def display_board_with_messages(top_message: nil, bottom_messages: [])
+    top_message ||= default_top_message
 
-  def display_board_with_messages(top_message: :title, bottom_messages: [])
     @game_messenger.display_board_with_messages(
       top_message: top_message,
       board: @board,
```

Comment:
@christophermanahan, you [asked me](https://github.com/k3ntako/super-tic-tac-toe/pull/13#discussion_r387304090) if this belonged in `GameState` in a separate PR.

I'm thinking that maybe it belongs here, because what it's doing is not displaying the messages/board, but instead asking `GameMessenger` to do that. `GameState.display_board_with_messages` is just taking all the information stored in its instance variables and passing them in.

I do think that `GameState` is getting big, and it has a growing number of dependencies. But, because all of their dependents rely on `GameState`'s state (i.e., current player, board state) it's not easy to break it apart. I think breaking it out would create tightly coupled classes and defeat the purpose of breaking out classes. 

This class is acting as a the central "brain" of the game. I thought this made things easier to understand. I would love to hear any feedback on this!